frontend/public/components/utils/__tests__/firehose.data.tsx

```diff
@@ -0,0 +1,42 @@
+import { Map as ImmutableMap } from 'immutable';
+
+export { PodModel } from '../../../models';
+
+export const podData = {
+  apiVersion: 'v1',
+  kind: 'Pod',
+  metadata: {
+    name: 'my-pod',
+    namespace: 'default',
+    resourceVersion: '123',
+  },
+};
+
+export const podList = {
+  apiVersion: 'v1',
+  kind: 'PodList',
+  items: ['my-pod1', 'my-pod2', 'my-pod3'].map((name) => ({
+    apiVersion: 'v1',
+    kind: 'Pod',
+    metadata: {
+      name,
+      namespace: 'default',
+      resourceVersion: '123',
+    },
+  })),
+  metadata: { resourceVersion: '123' },
+};
```

Comment:
we can reuse `podList` and `podData` from `useK8sWatchResource.data.tsx`. I think it is better to use `podList` and `podData` from here and delete `useK8sWatchResource.data.tsx` and `useK8sWatchResources.data.tsx`. wdyt?

Reply:
This is a different package. Yes, we have already dependencies from public package (console/internal) to  packages/console-dynamic-plugin-sdk but I would like to keep a copy here instead of adding more dependencies between both.